WPMedia/fusion-news-theme-blocks
# Added esc event listener to useEffect

## Description

The "Lead Art - Arc Block" fullscreen-exit button and the image-wrapper remain visible after I press the 'escape' key to exit from the fullscreen view of an image.

Sample page: https://cetest-cetest-sandbox.web.arc-cdn.net/2024/01/24/san-diego-wild-parrots/

## Jira Ticket

- [THEMES-1673](https://arcpublishing.atlassian.net/browse/THEMES-1673)

## Acceptance Criteria

Close button should not appear when image is not in fullscreen

## Test Steps

1. Checkout this branch `git checkout THEMES-1673-close-icon-fix`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/lead-art-block`
3.  Open page with a lead art image
4. Click the "expand" button to open the fullscreen view
5. Press the escape key
6. You should not notice close button

## Dependencies or Side Effects

_Examples of dependencies or side effects are:_

- Manifest pr for new block:
- Feature pack pr for local development:
- How to deploy to news: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138682964/News+Theme+2.0+Migration+Development+Deployment+Notes
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Dependency on another PR that needs to be merged first

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1673]: https://arcpublishing.atlassian.net/browse/THEMES-1673?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
